frontend/src/libs/components/image-selector/libs/hooks/libs/helpers/can-attach-cropper.helper.ts

```diff
@@ -0,0 +1,11 @@
+import { CROPPIE_CONTAINER_SIGNATURE } from '../constants/croppie-container-signature.constant.js';
+
+const checkCanAttachCropper = (
+  container: HTMLDivElement | null,
+): container is HTMLDivElement => {
+  return (
+    !!container && !container.classList.contains(CROPPIE_CONTAINER_SIGNATURE)
```

Comment:
```suggestion
Boolean(container) && !container.classList.contains(CROPPIE_CONTAINER_SIGNATURE)
```

Reply:
No, this won't work, We have already discussed similar case before. Boolean(container) does not narrow the actual type of container

Reply:
!container?.classList.contains(CROPPIE_CONTAINER_SIGNATURE) - won't it work the same then?